Tests/IntegrationTests/Fixtures/1Dimension/nodeTreePresets.e2e.js

```diff
@@ -40,20 +40,6 @@ const SETTINGS_WITH_NODE_TREE_PRESETS = {
                         presets: {
                             'default': {
                                 baseNodeType: 'Neos.Neos:Document,!Neos.TestNodeTypes:Document.Blog,!Neos.TestNodeTypes:Document.BlogArticle'
-                            },
-                            'blog': {
```

Comment:
i think we should try to keep it and make it work on chrome. I dont know why it doesnt work in ci. Locally it works splendidly. We have to debug the http request

Reply:
I had the issue also locally, and therefore I changed this and also did not see why we need to change it. But I was not involved in the change back then.